sections/collapsible-content.liquid

```diff
@@ -0,0 +1,383 @@
+{{ 'component-accordion.css' | asset_url | stylesheet_tag }}
+{{ 'collapsible-content.css' | asset_url | stylesheet_tag }}
+
+<div class="collapsible-content collapsible-{{ section.settings.layout | split: '_' | first }}-layout color-{{ section.settings.color_scheme }}{% if section.settings.layout == 'section_container' %} page-width{% endif %}">
```

Comment:
Instead of `image_first` and `text_first` as layout option values, we could just use `image` and `text` in the schema respectively? Then we could simplify to `collapsible-{{ section.settings.layout }}-layout`?

Reply:
Hmm this is using `section.settings.layout`. The values you're mentioning are for `settings.desktop_layout` so I think we can keep it as is.

Reply:
Ah yeah true! I feel like the values can still be simplified to `none`, `row`, and `section` to avoid `split` filter

Reply:
Good point 👍  done 👌